### PR TITLE
contig map should use alloc_domain for contiguity check

### PR DESCRIPTION
**Pull request by jjsjann123**

### CODE REVIEW

**Comment by jjsjann123** [comment]
!build

**Comment by jjsjann123** [comment]
sorry... clicked request for review by mistake... @zasdfgbnm 
I'll update the variable names and ping you for review afterwards.

File: csrc/scheduler/vectorize_helper.cpp [hunk 1]
```diff
@@ -690,36 +690,36 @@ void ContiguousInnerDimensionsMapper::propagateSibling(
 Val* ContiguousInnerDimensionsMapper::getContigMergeOfInnerSize(
     TensorView* of_tv) {
   Val* product_of_inner_extents = of_tv->container()->oneVal();
-  auto of_tv_root = of_tv->getMaybeRFactorDomain();
+  auto of_tv_alloc = of_tv->getMaybeAllocationDomain();
```
Comment (jjsjann123) [inline csrc/scheduler/vectorize_helper.cpp:693]: This line is the only real change. The rest is just renaming.

**Comment by jjsjann123** [comment]
!build

**Comment by jjsjann123** [comment]
This one is ready for review now. Just a one liner with some variable renaming. cc'ing @zasdfgbnm

**Review by zasdfgbnm — APPROVED** [review]

**Comment by jjsjann123** [comment]
Low risk item. merging since CI is looking reasonably green :laughing:

**Pull request merged**